### PR TITLE
[stable/external-dns] Add flags to extra args and version update to 0.4.4

### PR DESCRIPTION
**Pull request by s4nch3z**

### CODE REVIEW

File: stable/external-dns/README.md [hunk 1]
```diff
@@ -26,21 +26,21 @@ The following tables lists the configurable parameters of the external-dns chart
...
 | `aws.secret_key`          | `AWS_SECRET_ACCESS_KEY` to set in the environment (optional).                                                              | `""`                                               |
 | `aws.region`              | `AWS_DEFAULT_REGION` to set in the environment (optional).                                                                 | `us-east-1`                                        |
-| `cloudflare.api_key`      | `CF_API_KEY` to set in the environment (optional).                                                                 | `""`                                        |
-| `cloudflare.email`        | `CF_API_EMAIL` to set in the environment (optional).                                                                 | `""`                                        |
+| `cloudflare.api_key`      | `CF_API_KEY` to set in the environment (optional).                                                                         | `""`                                               |
```
Comment (lachie83) [inline stable/external-dns/README.md:29]: Remove EOL tabs

Comment (s4nch3z) [reply stable/external-dns/README.md:29]: What tabs? The file was formatted with spaces to look good in editor, except for those two line. So I made them match the rest of formatting.

Comment (viglesiasce) [reply stable/external-dns/README.md:29]: Yep just spaces

**Pull request merged**